tests/kafkatest/services/kafka/kafka.py

```diff
@@ -447,10 +449,29 @@ def create_topic(self, topic_cfg, node=None):
         self.logger.info("Running topic creation command...\n%s" % cmd)
         node.account.ssh(cmd)
 
-        time.sleep(1)
-        self.logger.info("Checking to see if topic was properly created...\n%s" % cmd)
-        for line in self.describe_topic(topic_cfg["topic"]).split("\n"):
-            self.logger.info(line)
+        if describe:
```

Comment:
Do you think this check is useful in general? I'm sort of inclined to just get rid of the logic below.

Reply:
I left this mainly for compat with existing tests that might rely on the output of this call (not sure if there are any). I'm fine removing it though

Reply:
Yeah, I think we should remove it. I don't know of any tests which rely on the test logging itself.